ballerina/build.gradle

```diff
@@ -127,6 +127,7 @@ task startBallerinaSTS() {
         }
     }
 }
+startBallerinaSTS.onlyIf { !project.gradle.startParameter.excludedTaskNames.contains('test') }
```

Comment:
This might introduce a dependency issue since the build is set to finalize by `startBallerinaSTS`. I think this approach is better:
https://github.com/ballerina-platform/module-ballerina-oauth2/pull/895